lib/NfcoreTemplate.groovy

```diff
@@ -146,10 +165,10 @@ class NfcoreTemplate {
     }
 
     //
-    // Construct and send adaptive card
-    // https://adaptivecards.io
+    // Construct and send a notification to a web server as JSON
+    // e.g. Microsoft Teams and Slack
     //
-    public static void adaptivecard(workflow, params, summary_params, projectDir, log) {
+    public static void IM_notification(workflow, params, summary_params, projectDir, log) {
```

Comment:
Is this new in the Nextflow template? And did you test this?

Reply:
yup and nope...i'm not even sure how I would go about testing this